src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -156,27 +150,37 @@ export class Registration extends React.Component<RegistrationProps, Registratio
   createBidder() {
     commitMutation(this.props.relay.environment, {
       onCompleted: (results, errors) =>
-        isEmpty(errors) ? console.log("COMPLETED!", results) : this.presentErrorResult(errors, msgForGraphQLError),
-      onError: error => this.presentErrorResult(error, msgForNetworkError),
+        isEmpty(errors)
+          ? this.presentRegistrationSuccess(results)
+          : this.presentRegistrationError(errors, RegistrationStatus.RegistrationStatusError),
+      onError: error => {
+        this.presentRegistrationError(error, RegistrationStatus.RegistrationStatusNetworkError)
+      },
       mutation: bidderMutation,
       variables: { input: { sale_id: this.props.sale.id } },
     })
   }
 
-  presentErrorResult(error, message) {
-    console.error(error)
+  presentRegistrationSuccess({ createBidder }) {
+    const qualifiedForBidding = createBidder.bidder.qualified_for_bidding
+    if (Boolean(qualifiedForBidding) === true) {
```

Comment:
Is it necessary to wrap `qualifiedForBidding` with a `Boolean(...)`?

Reply:
Oh I guess not!